test_my_changes.py

```
@@ -0,0 +1,55 @@
+        with warnings.catch_warnings(record=True) as w:
+            warnings.simplefilter("always")
+            loader = LoadImage(reader="UnknownReader", raise_on_missing_reader=False)
+            if w and "UnknownReader" in str(w[0].message):
+                print(f"PASS: Got expected warning: {w[0].message}")
+            else:
+                print("WARNING: Warning message may have been different than expected")
+        print("PASS: LoadImage instance created successfully with fallback behavior")
+    except OptionalImportError as e:
+        print(f"FAIL: Unexpected OptionalImportError with flag disabled: {e}")
+        return False
+    except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)